src/uct/api/uct.h

```diff
@@ -332,8 +332,11 @@ typedef struct uct_tl_resource_desc {
                                                 (e.g. UCT_DEVICE_TYPE_NET for a network interface) */
     ucs_sys_device_t         sys_device;   /**< The identifier associated with the device
                                                 bus_id as captured in ucs_sys_bus_id_t struct */
+    uint8_t                  flags;        /**< Associated flags to the resource */
 } uct_tl_resource_desc_t;
 
+#define UCT_TL_RESOURCE_FLAG_INTER_NODE UCS_BIT(0) /**< Inter-node capability */
```

Comment:
Does it make sense to declare a enum with name, and reference that name in the `flags` field documentation?
Otherwise it's hard to understand afterwards which enum/values are supposed to be set